### DISCUSSION TITLE
How to add new different planets

### DISCUSSION THREAD

**Discussion by greenmughal** [fintelia/terra#22] [General]
Can you give an outline how to add new different planets, what is the process?

**Comment by fintelia** [comment]
What planets are you hoping to add?

The bulk of the effort would be in acquiring and reprojecting [heightmaps](https://github.com/fintelia/terra/blob/afdd9fb6caea2128aba409f30709b3e2ce8f7cbd/src/generate/mod.rs#L642) and [albedo information](https://github.com/fintelia/terra/blob/afdd9fb6caea2128aba409f30709b3e2ce8f7cbd/src/generate/mod.rs#709) for the planet. For Earth, terra uses the SRTM dataset and NASA's Blue Marble: Next Generation respectively.
